src/Components/WeekdayCardsOnline/WeekdayCardsOnline.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./WeekdayCardsOnline.module.css";
+
+function WeekdayCardsOnline() {
+  return (
+    <div className={styles.cards}>
+      <a href="/monday-online">
+        <div className={styles.singlecard}>
```

Comment:
You might not need this div to get to the same visual result when styling the `h4` elements directly. If you are curious, you could try it.